packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/kubelet.rules.yaml.patch

```diff
@@ -10,8 +10,8 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -45,4 +46,5 @@
-         {{ toYaml .Values.defaultRules.additionalRuleLabels | nindent 8 }}
```

Comment:
Same comment: do we need to keep these?

Reply:
See above: https://github.com/rancher/charts/pull/3930#discussion_r1611735165